### DISCUSSION TITLE
Vercel : Rewriting rules 404 issue

### DISCUSSION THREAD

**Discussion by MacMah0n** [vercel/next.js#21204] [Help] [answered]
Hello !

I have an error on vercel that i haven't on local.
The deployment is successful but don't works as expected.
This example is here : https://rewrite-exemple-error.vercel.app/

You need to navigate to the Example page -> it's works. 
But if you do a refresh, a 404 is showed. On local it's works perfectly.
I have a custom rewrite rule in my next.config.js file, but nothing complicated...

There is a public repo with the simple exemple : https://gitlab.com/macmah0n/rewrite-example-error 

Thanks !

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like you're using Next.js v10.0.1, there were some patches to rewrites + i18n in the latest release `v10.0.5`, please update and give it a try!

**Reply by MacMah0n** [reply]
Hello @ijjk 

Thank you, it works perfectly :)
